lib/python/picongpu/plugins/plot_mpl/base_visualizer.py

```diff
@@ -0,0 +1,44 @@
+import matplotlib.pyplot as plt
+
+
+class Visualizer(object):
+    def __init__(self, run_directory):
+
+        if run_directory is None:
+            raise ValueError('The run_directory parameter can not be None!')
+
```

Comment:
maybe add a filesystem check if the dir exists?

Reply:
Isn't it sufficient that the reader performs this check? From what I saw they all check for existence in their constructors.